chia/wallet/did_wallet/did_wallet.py

```diff
@@ -1319,7 +1319,7 @@ def deserialize_backup_data(backup_data: str) -> DIDInfo:
         :return: DIDInfo
         """
         details = backup_data.split(":")
-        origin = Coin(bytes32(bytes.fromhex(details[0])), bytes32(bytes.fromhex(details[1])), uint64(int(details[2])))
+        origin = Coin(bytes32.fromhex(details[0]), bytes32.fromhex(details[1]), uint64(int(details[2])))
```

Comment:
I think this should be `from_hexstr` and not `fromhex`.  Pretty sure this code is not in use so it doesn't matter but if I try to from `bytes32.fromhex("abcdef")` I get `ValueError: bad bytes32 initializer b'\xab\xcd\xef'`.

Reply:
you mean because the strings start with `0x`? if it isn't used, we really should remove it though

Reply:
I believe the error you got was because the hex string wasn't 32 bytes long. My understanding of the difference between `fromhex()` and `from_hexstr()` is that the latter expects a `0x` prefix

Reply:
Oh yeah whoops you're right :)